### ISSUE TITLE
Java buildpack fails because of fetch returning HTTP 404

### ISSUE THREAD

**Issue by mkolodziejczak** [GoogleCloudPlatform/buildpacks#421]
I can't build my Java Cloud Run app because one of the fatches that happens during the build returns 404 error.

Namely:
Step #0 - "Buildpack": fetching https://dl.google.com/runtimes/ubuntu1804/canonicaljdk/version.json returned HTTP status: 404

**Comment by kennethye1** [comment]
I do not believe we support Java 21 on `builder:v1`. You can use `builder:google-22`.
